BoltsTests/AppLinkReturnToRefererViewTests.m

```diff
@@ -74,6 +74,24 @@ - (void)testValidRefererDataResultsInNonZeroSizeThatFits {
     XCTAssert(sizeThatFits.width > 0.0);
 }
 
+- (void)testIncludesStatusBarResultsInLargerHeight {
+    NSURL *url = [NSURL URLWithString:BFURLWithRefererData];
+    BFAppLink *appLink = [[BFURL URLWithURL:url] appLinkReferer];
+
+    BFAppLinkReturnToRefererView *view = [[BFAppLinkReturnToRefererView alloc] init];
+    view.refererAppLink = appLink;
+    view.includeStatusBarInSize = BFIncludeStatusBarInSizeNever;
+
+    CGSize sizeThatFitsNotIncludingStatusBar = [view sizeThatFits:CGSizeMake(100.0, 100.0)];
+
+    view.includeStatusBarInSize = BFIncludeStatusBarInSizeAlways;
+
+    CGSize sizeThatFitsIncludingStatusBar = [view sizeThatFits:CGSizeMake(100.0, 100.0)];
+
+    XCTAssert(sizeThatFitsIncludingStatusBar.height > sizeThatFitsNotIncludingStatusBar.height);
+}
+
+
```

Comment:
Somewhat. There are 2 blank lines here, compared to a single line between every other test here.